### ISSUE TITLE
Search for keywords

### ISSUE THREAD

**Issue by schiessle** [nextcloud/integration_discourse#4]
It would be nice if I could define keywords I'm interested in and the app would search for posts including the keyword and add them to the dashboard.

**Comment by eneiluj** [comment]
Yeah i had the same idea with following hashtags in Mastodon and Twitter integrations.

The current way of declaring a widget might not allow having multiple instances of the same widget. I'll check this out. This is definitely a good idea.

Thanks for the feedback.

**Comment by schiessle** [comment]
Multiple instances of the same widget might be nice in general. Also for people who are active in multiple Discourse boards, Gitlab instances, Mastodon servers, etc.

For the keyword search I think the results could go in the same dashboard widget as the normal notifications.

**Comment by adamshand** [comment]
This would be very cool!
